packages/design-system/src/components/sidebar/useSidebar/index.tsx

```diff
@@ -16,7 +16,15 @@
 /**
```

Comment:
Can we split this as well into:
- `context.ts`
- `sidebarProvider.tsx`
- `useSidebar.ts`

Reply:
@mayan-000 Please resolve the requested change.